Sources/Danger/Danger.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 // MARK: - DangerRunner
 
 private final class DangerRunner {
-    let version = "0.3.6"
+    let version = "0.3.7"
 
     static let shared = DangerRunner()
 
```

Comment:
<!--
  0 failure: 
  1 warning:  Hello world3
  
  
  DangerID: danger-id-default;
  File: Sources/Danger/Danger.swift;
  Line: 15;
-->  

- :warning: Hello world3

Reply:
cc @sunshinejr 🤣

Reply:
It works! 😆